src/Microsoft.AspNetCore.Mvc.Abstractions/ModelBinding/ModelMetadata.cs

```diff
@@ -43,6 +43,17 @@ protected ModelMetadata(ModelMetadataIdentity identity)
         public Type ContainerType { get { return Identity.ContainerType; } }
 
         /// <summary>
+        /// Gets the metadata of the container type that the current instance is part of.
+        /// </summary>
+        public virtual ModelMetadata ContainerMetadata
+        {
+            get
+            {
+                return null;
```

Comment:
`NotImplementedException` here too. Want to ensure subclasses override this non-implementation.